src/utils/form-ui.ts

```diff
@@ -87,11 +87,28 @@ function getPredicateName(predicate: string): any {
   return null
 }
 
-async function getPropertyValue(field: string, property: string) {
+/**
+ * Fetch a value for a given subject and property, with special considerations for specific properties
```

Comment:
Haha, I'm happy there's a bit of documentation here, but I really have no idea what "special considerations" and "specific properties" are :P